product/domain/productBasics_test.go

```diff
@@ -80,7 +80,8 @@ func TestAttributeValues(t *testing.T) {
 	a := Attribute{RawValue: "some string"}
 	result := a.Values()
 	assert.IsType(t, []string{}, result)
-	assert.Empty(t, result)
+	assert.Len(t, result, 1)
+	assert.Equal(t, "some string", result[0])
```

Comment:
please add this kind of assertion also for the slice case

Reply:
updated the test please have a look